src/gui/static/e2e/wallets.po.ts

```diff
@@ -0,0 +1,231 @@
+    const seedEl = element(by.css('[formcontrolname="seed"]'));
+    const btn = element(by.buttonText(confirm ? 'Create' : 'Load'));
+
+    labelEl.clear();
+    seedEl.clear();
+    labelEl.sendKeys(label);
+    seedEl.sendKeys(seed);
+
+    if (confirm) {
+      const confirmEl = element(by.css('[formcontrolname="confirm_seed"]'));
+      confirmEl.clear();
+      confirmEl.sendKeys(seed);
+    }
+
+    return btn.isEnabled();
+  }
+
+  createWallet() {
+    const label = element(by.css('[formcontrolname="label"]'));
+    const seed = element(by.css('[formcontrolname="seed"]'));
+    const confirm = element(by.css('[formcontrolname="confirm_seed"]'));
+    const encrypt = element(by.css('.mat-checkbox-label'));
+    const btnCreate = element(by.buttonText('Create'));
+
+    label.clear();
+    label.sendKeys('Test create wallet');
+    seed.clear();
+    seed.sendKeys('test create wallet');
+    confirm.clear();
+    confirm.sendKeys('test create wallet');
+    encrypt.click();
+
+    return btnCreate.isEnabled().then(status => {
+      if (status) {
+        btnCreate.click();
+      }
+
+      return status;
+    });
+  }
+
+  loadWallet() {
+    const label = element(by.css('[formcontrolname="label"]'));
+    const seed = element(by.css('[formcontrolname="seed"]'));
+    const encrypt = element(by.css('.mat-checkbox-label'));
+    const btnLoad = element(by.buttonText('Load'));
+
+    label.clear();
+    label.sendKeys('Test load wallet');
+    seed.clear();
+    seed.sendKeys('test load wallet');
+    encrypt.click();
+
+    return btnLoad.isEnabled().then(status => {
+      if (status) {
+        btnLoad.click();
+      }
+
+      return status;
+    });
+  }
+
+  expandWallet() {
+    return this.getWalletWithName('Test create wallet').click().then(() => {
+      return element(by.css('app-wallet-detail')).isPresent();
+    });
+  }
+
+  showQrDialog() {
+    return browser.sleep(1000).then(() => element(by.css('.qr-code-button')).click().then(() => {
+      return element(by.css('app-qr-code')).isPresent();
+    }));
+  }
+
+  hideQrDialog() {
+    return browser.sleep(1000).then(() => element(by.css('app-modal .-header img')).click().then(() => {
+      return element(by.css('app-qr-code')).isPresent();
+    }));
+  }
+
+  addAddress() {
+    return element.all(by.css('.-detail')).count().then(originalCount => {
+      return element(by.css('.-new-address')).click().then(() => {
+        return browser.sleep(2000).then(() => {
+          return element.all(by.css('.-detail')).count().then(newCount => {
+            return newCount > originalCount;
+          });
+        });
+      });
+    });
+  }
+
+  hideEmptyAddress() {
+    return element(by.css('.-hide-empty')).click().then(() => {
+      return element.all(by.css('.-detail > div:nth-child(3)')).filter((address) => {
+        return address.getText().then(value => {
+          return value === '0';
+        });
+      }).count();
+    });
+  }
+
+  showEmptyAddress() {
```

Comment:
This function should be very similar to `hideEmptyAddress`, because `hideEmptyAddress` counts the number of addresses without coins that are shown. Even the two functions could be one, since the only difference would be the selector.